### PR TITLE
feat: initial french translation

### PR DESCRIPTION
**Pull request by firebomb1**
## Description

Translation of the UI strings in French.

## Notice

Before submitting a pull request, please make sure you have answered the following:

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

### CODE REVIEW

**Comment by LPX55** [comment]
https://blockscout.com/xdai/mainnet/tx/0xc1382f15122c788c400837aae25847c182b8f8cf8336e126141b072ee8983507

File: src/assets/translations/messages-fr.json [hunk 1]
```diff
@@ -0,0 +1,531 @@
...
+    "error": "Erreur",
+    "cancel": "Annuler",
+    "continue": "Continuer",
+    "confirm": "Confirmer",
+    "reject": "Rejeter",
+    "connected": "Connecté",
+    "created": "Créé le %{date}",
+    "noResultsFound": "Aucun résultat trouvé.",
+    "insufficientFunds": "Fonds insuffisants",
+    "invalidAddress": "Adresse non valable",
```
Comment (gomesalexandre) [inline src/assets/translations/messages-fr.json:14]: non valable -> non valide / invalide ? 🤔

File: src/assets/translations/messages-fr.json [hunk 2]
```diff
@@ -0,0 +1,531 @@
...
+    "unresolvableEnsDomain": "Impossible de résoudre le domaine ENS",
+    "deposit": "Déposer",
+    "withdraw": "Retirer",
+    "getStarted": "Commencez",
```
Comment (gomesalexandre) [inline src/assets/translations/messages-fr.json:18]: "Commencer" - all other CTAs use infinitive

File: src/assets/translations/messages-fr.json [hunk 3]
```diff
@@ -0,0 +1,531 @@
...
+    "somethingWentWrongBody": "Il y a eu un problème",
+    "transactionFailed": "Échec de la transaction",
+    "transactionFailedBody": "Échec de la transaction",
+    "submitFeedback": "Envoyer un commentaire",
+    "send": "Envoyer",
+    "receive": "Recevoir"
+  },
+  "updateToast": {
+    "body": "Une nouvelle version de l'app est disponible",
+    "cta": "Recharger l'App"
```
Comment (gomesalexandre) [inline src/assets/translations/messages-fr.json:37]: Recharger -> Rafraichir

File: src/assets/translations/messages-fr.json [hunk 4]
```diff
@@ -0,0 +1,531 @@
...
+  },
+  "dashboard": {
+    "portfolio": {
+      "addAsset": "Ajouter Actif",
+      "yourAssets": "Vos Actifs",
+      "asset": "Actif",
+      "portfolioBalance": "Solde du Portefolio",
```
Comment (gomesalexandre) [inline src/assets/translations/messages-fr.json:44]: Portefeuille?

File: src/assets/translations/messages-fr.json [hunk 5]
```diff
@@ -0,0 +1,531 @@
...
+    "confirmAndTrade": "Confirmer et échanger",
+    "complete": "Échange terminé",
+    "approveAsset": "Approuver l'interaction avec %{symbol}",
+    "needPermission": "Nous avons besoin de votre permission pour utiliser votre %{symbol}",
+    "whyNeedThis": "Pourquoi dois-je faire ça ?",
+    "estimatedGasFee": "Estimation des frais de carburant",
+    "approvingAsset": "Approbation de %{symbol}",
+    "viewTransaction": "Voir la transaction",
+    "tooltip": {
+      "rate": "C'est le taux attendu pour cette paire d'échanges.",
```
Comment (gomesalexandre) [inline src/assets/translations/messages-fr.json:158]: attendu -> estimé?

File: src/assets/translations/messages-fr.json [hunk 6]
```diff
@@ -0,0 +1,531 @@
...
+      "verify": "Vérifier",
+      "verified": "Vérifié",
+      "notVerified": "Mauvaise concordance d'adresse",
+      "blockExplorer": "Explorateur"
+    },
+    "deposit": {
+      "amountToDeposit": "Montant à déposer",
+      "estimatedReturns": "Rendement annuel estimé*",
+      "estimateDisclaimer": "*Les estimations fluctuent en fonction des conditions du marché.",
+      "footerDisclaimer": "En continuant, vous acceptez nos conditions de service.",
```
Comment (gomesalexandre) [inline src/assets/translations/messages-fr.json:270]: conditions de service -> conditions d'utilisation

**Review by gomesalexandre — APPROVED** [review]
A bunch of copy improvements but I suppose for now the goal is mostly to match the current copies' keys 1/1, not to have perfect translations?

Is there any place where these were originally contributed and commenting would make more sense? I remember seeing something on oneDAO but I reckon I didn't have access to that platform that's used for translations.

**Comment by LPX55** [comment]
@gomesalexandre  It was requested of us to make a PR directly to the main develop branch instead of going through a QC period on our end first where we get third party reviewers on certain occasions, and I certainly think this occasion warrants a third party check on all languages. There's going to be more PRs in the hours to come for other languages, I guess you can view them as draft PRs. Sorry about the mess!

Thanks for all your input btw, super helpful. Maybe we can skip French peer review now ;) (half-kidding)

**Comment by gomesalexandre** [comment]
> @gomesalexandre I guess you can view them as draft PRs. Sorry about the mess!
> 
> 
> 
> Thanks for all your input btw, super helpful. Maybe we can skip French peer review now ;) (half-kidding)

That's what I got from the looks of this PR yeah! If this needs a second pair of eyes, I'll be here on the follow-up PRs :)

**Comment by firebomb1** [comment]
Thanks for the suggestions @gomesalexandre, I've applied all of them.

**Review by gomesalexandre — APPROVED** [review]
Thanks for updating copies!

**Review by 0xdef1cafe — APPROVED** [review]

**Comment by 0xdef1cafe** [comment]
me reviewing this PR

https://www.tiktok.com/@westonapplegate/video/6771580231741410565?is_from_webapp=1&sender_device=pc&web_id7061362394286605830

**Pull request merged**